setup.cfg

```diff
@@ -1,10 +1,10 @@
 [flake8]
-ignore = E731, E402, W504
+ignore = E731, E402, W504, C814
```

Comment:
are you use 814 is the right number. In the docs of the flake8 package it [has](https://github.com/PyCQA/flake8-commas#errors) many similar errors listed and the most generic one sounds like 812